lib/hooks/useClient.js

```diff
@@ -4,12 +4,12 @@ import { MeiliSearch } from "meilisearch";
 export const useClient = ({ uid } = {}) => {
   if (typeof window === "undefined") return {};
   const {
-    keys: { private: privateKey },
     host,
+    key,
   } = useStore();
   const searchClient = new MeiliSearch({
     host,
-    apiKey: privateKey,
```

Comment:
Can you tell me why you changed this?

Reply:
**_useStore_** function returns this object:
`{
    "host": "http://111.111.111.111:7700",
    "key": "XXXXXXXXXXXXXXXXXXXXXXXXX",
    "keys": {
        "message": "The Authorization header is missing. It must use the bearer authorization method.",
        "code": "missing_authorization_header",
        "type": "auth",
        "link": "https://docs.meilisearch.com/errors#missing_authorization_header"
    }
}`
and using _**privateKey**_, I was having an error because _**apiKey**_ was being sent empty, then I've changed to _**key**_ and works fine